code/datums/bodypart_overlays/simple_bodypart_overlay.dm

```diff
@@ -11,7 +11,10 @@
 /datum/bodypart_overlay/simple/get_image(layer, obj/item/bodypart/limb)
 	return image(icon, icon_state, layer = layer)
 
-/datum/bodypart_overlay/simple/color_image(image/overlay, layer)
+/datum/bodypart_overlay/simple/color_image(image/overlay, layer, obj/item/bodypart/limb)
+	if(!overlay)
+		return
```

Comment:
Just let this one runtime tbh. I don’t think you can even break this one

Reply:
Yeah, you can break it, if the one before early-returns due to the crash, this one is going to runtime too. Which is kind of useless information since we already have the crash on the proc that gets called before it.

Reply:
> Yeah, you can break it, if the one before early-returns due to the crash, this one is going to runtime too. Which is kind of useless information since we already have the crash on the proc that gets called before it.

Never mind you're right.